packages/office-addin-manifest/src/commands.ts

```diff
@@ -92,7 +92,9 @@ function logManifestValidationWarnings(warnings: ManifestValidationIssue[] | und
 }
 
 function logManifestValidationIssue(issue: ManifestValidationIssue) {
-  console.log(`${issue.title}: ${issue.content}` + (issue.helpUrl ? ` (link: ${issue.helpUrl})` : ``));
+  console.log(
+    (issue.title ? `${issue.title}: ` : ``) + `${issue.content}` + (issue.helpUrl ? ` (link: ${issue.helpUrl})` : ``)
+  );
```

Comment:
What are the cases where the title is going to be empty?

Reply:
None! I will remove this, as I'm already adding titles to the json validation